bundle.Dockerfile

```diff
@@ -1,11 +1,43 @@
 FROM scratch
 
-LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
-LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
-LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
-LABEL operators.operatorframework.io.bundle.package.v1=windows-machine-config-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
-
-COPY deploy/olm-catalog/windows-machine-config-operator/manifests /manifests/
-COPY deploy/olm-catalog/windows-machine-config-operator/metadata /metadata/
+COPY manifests /manifests/
+COPY metadata/annotations.yaml /metadata/annotations.yaml
```

Comment:
Please add a comment here saying that these directories maps to the layout of the dist-git repo. Or we need to change the dist-git repo to use `deploy/olm-catalog/windows-machine-config-operator/` and update `container.yaml` with that.